sdks/python/src/opik/api_objects/attachment/attachment.py

```diff
@@ -0,0 +1,15 @@
+import dataclasses
+from typing import Optional
+
+
+@dataclasses.dataclass
+class Attachment:
+    """Represents an Attachment to be added to the Trace or Span.
+    Args:
+        data: The data to be added to the Attachment.
```

Comment:
Let's have a more descriptive docstring for `data`, it's quite hard to understand that you should pass a file path to it.